bin/tag-release.sh

```diff
@@ -19,9 +19,25 @@ while [[ $# -ge 1 ]]; do
   shift # past argument or value
 done
 
+# compare master and stage
+echo "Comparing master branch to staging..."
+git fetch "$moz_git_remote"
+stage_hash=`git rev-parse $moz_git_remote/stage`
+master_hash=`git rev-parse master`
+if [ $stage_hash != $master_hash ]; then
+    read -p "Master branch does NOT match stage branch! Are you sure you want to continue? (Type Override to continue, n to cancel)" no_match
+    if [ "$no_match" == "${no_match#[Override]}" ]; then
```

Comment:
I think this should just be:

```suggestion
    if [[ "$no_match" == "Override" ]]; then
```

Reply:
Or if you want/need to only check that it starts with "Override" then this would work:

```suggestion
    if [[ "$no_match" == Override* ]]; then
```

Reply:
I tried both suggestions but both continued when I typed no :/

Reply:
Huh. Oh well. Sorry about that :/